.gitignore

```diff
@@ -40,6 +42,7 @@ pip-delete-this-directory.txt
 htmlcov/
 .tox/
 .coverage
+.coveragerc
```

Comment:
@mwinel this file doesn't exist in the code base. Remove this line.

Reply:
We need it incase we are trying to execute a coverage report locally.